Added physical references for special fax networks

Kocma-san opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Now there are fax machines that receive and send messages on behalf of the special networks of NT and syndicate
Also added centcom stamps to centcom (funny, they weren’t there)

![image](https://github.com/user-attachments/assets/e4330793-c87b-4d99-9a86-da795969335b)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Faxes sent from the CC will have the correct sender name. Also, there will be no unnecessary faxes in the fax panel

<!-- Argue for the merits of your changes and how they benefit the game, especially if they are controversial and/or far reaching. If you can't actually explain WHY what you are doing will improve the game, then it probably isn't good for the game in the first place. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
code: сhanges for a fax located on the CC
qol: added centcom stamps to centcom
/:cl

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Kocma-san committed:
probably

SyncIt21 commented on code/modules/admin/admin_fax_panel.dm:
```diff
@@ -68,7 +68,7 @@ ADMIN_VERB(fax_panel, R_ADMIN, "Fax Panel", "View and respond to faxes sent to C
 	for(var/datum/weakref/weakrefed_fax as anything in available_faxes)
 		var/obj/machinery/fax/another_fax = weakrefed_fax.resolve()
-		if(another_fax && istype(another_fax))
+		if(another_fax && istype(another_fax) && (another_fax.fax_id != another_fax.special_networks["nanotrasen"]["fax_id"]) && (another_fax.fax_id != another_fax.special_networks["syndicate"]["fax_id"]))
```
Please find a better way. This if statement is way too snowflacky

Kocma-san committed:
can do it this way only

but I don't like it

Kocma-san committed:
done

And now I don't dislike it

Kocma-san committed:
Update fax.dm

Kocma-san committed:
Update fax.dm

Ghommie commented:
merge conflicts

Kocma-san committed:
Merge remote-tracking branch 'upstream/master' into some-changes-on-centcom

Kocma-san committed:
Changes

Functionality is the same as PR #84885, but with the following additions/changes:
- Admins faxes are no longer displayed in the fax panel, and also have the correct names and IDs
- Letters sent to the syndicate are no longer printed on the Central Committee
- Changed the location of the fax on the CC

Kocma-san committed:
fix

Kocma-san commented:
Not sure what to do with letters sent to a special syndicate network. They are not printed on the CС fax, but the syndicate fax is not located anywhere...
